libevmcore/Instruction.h

```diff
@@ -47,6 +47,9 @@ enum class Instruction: uint8_t
 	EXP,                ///< exponential operation
 	SIGNEXTEND,         ///< extend length of signed integer
 
+	RETURNDATASIZE = 0x0d,  ///< size of data returned from previous call
+	RETURNDATACOPY = 0x0e,  ///< copy data returned from previous call to memory
```

Comment:
The opcodes seem to have changed: https://github.com/ethereum/solidity/pull/2275#discussion_r116934325